.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,124 @@
+name: 🐛 Bug report
+description: Create a report to help us improve.
+labels:
+  - Bug
+
+body:
+  - type: textarea
+    attributes:
+      label: Bug Description
+      description: Please describe clearly and concisely what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected Behaviour
+      description: Please describe clearly and concisely what the expected behaviour should be.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: >
+        Please provide detailed steps on how to reproduce the bug. Provide a URL where the issue can be seen on the
+        frontend when possible, otherwise go to “View source” in the browser and copy all to paste in a
+        [Gist](https://gist.github.com/) and share it.
+
+  - type: input
+    attributes:
+      label: PHP Version
+      description: Specify which PHP version the bug occurs on.
+      placeholder: e.g. 8.1
+    validations:
+      required: true
```

It may not be relevant to PHP, as it could be a JS bug.

```suggestion
      required: false
```